hoot-core/src/main/cpp/hoot/core/cmd/ConvertCmd.cpp

```diff
@@ -78,10 +82,25 @@ class ConvertCmd : public BaseCommand
     OsmMapReaderFactory readerFactory = OsmMapReaderFactory::getInstance();
     OsmMapWriterFactory writerFactory = OsmMapWriterFactory::getInstance();
 
+    if (args[0].endsWith(".osc"))
+    {
+      throw HootException("XML changeset file writing is not supported by the convert command.");
+    }
+    else if (args[0].endsWith(".osc.sql"))
```

Comment:
I did.  I went back and forth on whether to create a new command for it or not.  Ended up putting it 'convert'.  Guessing from your response it should probably be in its own command :-)

I could add 'apply-changeset'.  I'm feeling really weird security-wise about exposing a command that's writes sql (happens with either command), but maybe I'm being too paranoid.  I wonder if 'derive-changeset' and 'apply-changeset' should be combined together automatically for sql changeset, thus not allowing any suspect sql to actually be passed in.